app/tests/unit/components/transformer.spec.js

```diff
@@ -0,0 +1,135 @@
+const helper = require('../../common/helper');
+const transformer = require('../../../src/components/transformer');
+
+helper.logHelper();
+
+describe('toStatusResponse', () => {
+  it('should return an empty object if all properties are undefined', () => {
```

Comment:
Need test for undefined, and "bad" object types (string, number, array)

Reply:
Implemented